shell/components/ResourceDetail/index.vue

```diff
@@ -209,13 +212,23 @@ export default {
       if (realMode === _VIEW) {
         model = liveModel;
       } else {
-        model = await store.dispatch(`${ inStore }/clone`, { resource: liveModel });
+        try {
+          model = await store.dispatch(`${ inStore }/clone`, { resource: liveModel });
+        } catch (e) {
+          this.errors.push(e);
+        }
+      }
+      try {
+        initialModel = await store.dispatch(`${ inStore }/clone`, { resource: liveModel });
+      } catch (e) {
+        this.errors.push(e);
       }
-
-      initialModel = await store.dispatch(`${ inStore }/clone`, { resource: liveModel });
-
       if ( as === _YAML ) {
-        yaml = await getYaml(this.$store, liveModel);
+        try {
+          yaml = await getYaml(this.$store, liveModel);
+        } catch (e) {
+          this.errors.push(e);
+        }
```

Comment:
It looks like we can wrap all of these statements into a single `try...catch` block:

```
      try {
        if (realMode === _VIEW) {
          model = liveModel;
        } else {
          model = await store.dispatch(`${ inStore }/clone`, { resource: liveModel });
        }

        initialModel = await store.dispatch(`${ inStore }/clone`, { resource: liveModel });

        if ( as === _YAML ) {
          yaml = await getYaml(this.$store, liveModel);
        }
      } catch (e) {
        this.errors.push(e);
      }
```